src/libs/fetchPronouns.js

```diff
@@ -172,10 +173,8 @@ function getPronounField(status, accountName) {
 	const fields = account.fields;
 
 	for (const field of fields) {
-		//match fields against "pronouns"
-		//TODO: multiple languages -> match against list
-		if (field.name.toLowerCase().includes("pronouns")) {
-			//TODO: see https://github.com/ItsVipra/ProToots/issues/28
+		//match fields against fieldNames
+		if (fieldNames.includes(field.name.toLowerCase())) {
```

Comment:
Note: matches less than before in some instances, i.E. partial fieldname matches (ex: "pronouns EN")